cmd/main.go

```diff
@@ -77,6 +77,11 @@ func Main() {
 	os.Exit(exitCode)
 }
 
+// GetCommand is provided to allow tests to extract the commands
+func GetCommand(name string) Command {
```

Comment:
@mholt this is pretty cool, change running the app locally, but I did hit a snag getting hold of the run command. So this is my work around.

It is called here.
https://github.com/caddyserver/caddy/blob/62d86d892b0c9c7865903c6d2b9771a72c3eef35/caddytest/caddytest.go#L151

Reply:
Instead, can you just manipulate `os.Args`? I know that's normally a no-no, but these are integration tests after all...